rtos/source/TARGET_CORTEX/mbed_rtos_rtx.c

```diff
@@ -32,6 +32,13 @@
 #include "FEATURE_TFM/interface/include/tfm_ns_lock.h"
 #endif
 
+#if defined(TARGET_TFM_TWINCPU)
+#include "FEATURE_PSA/TARGET_TFM/TARGET_TFM_TWINCPU/inc/tfm_multi_core_api.h"
```

Comment:
The build tool will automatically add the include path `FEATURE_PSA/TARGET_TFM/TARGET_TFM_TWINCPU/inc/`. It isn't a good idea to hardcode the include path. Same applies to other headers as well.